public/video-ui/src/actions.ts

```diff
@@ -0,0 +1,60 @@
+import { PlutoCommission, PlutoProject } from "./services/PlutoApi";
+import { Video } from "./services/VideosApi";
+
+
+type ShowError = {
+    type: 'SHOW_ERROR',
+    message: string,
+    receivedAt: number,
+    error: unknown
+}
+
+type PlutoCommissionsGetRequest = {
+    type: 'PLUTO_COMMISSIONS_GET_REQUEST',
+    receivedAt: number,
+}
+
+type PlutoCommissionsGetReceive = {
+    type: 'PLUTO_COMMISSIONS_GET_RECEIVE',
+    receivedAt: number,
+    commissions: PlutoCommission[]
+}
+
+type PlutoProjectGetRequest = {
+    type: 'PLUTO_PROJECTS_GET_REQUEST',
+    receivedAt: number,
+}
+
+type PlutoProjectsGetReceive = {
+    type: 'PLUTO_PROJECTS_GET_RECEIVE',
+    receivedAt: number,
+    projects: PlutoProject[],
+}
+
+type AddProjectRequest = {
+    type: 'ADD_PROJECT_REQUEST',
+    receivedAt: number,
+}
+
+type AddProjectReceive = {
+    type: 'ADD_PROJECT_RECEIVE',
+    receivedAt: number,
+    video: Video,
+}
+
+/**
+ * A union of the Action types that dispatched from ts tiles.
+ * 
+ * This is not yet a complete list of all the actions the application handles as
+ * some actions dispatched from js files.
+ */
+type KnownAction =
```

Comment:
Is it worth updating this type to capture that `receivedAt` is defined on every action? I can see the argument that we haven’t typed every action yet, but I’d be quite tempted by the reduction in duplication.